dev/provider_packages/prepare_provider_packages.py

```diff
@@ -1088,6 +1078,60 @@ def get_additional_package_info(provider_package_path: str) -> str:
     return ""
 
 
+def get_changelog_for_package(provider_package_path: str) -> str:
+    """
+    Returns changelog_for the package.
+
+    :param provider_package_path: path for the package
+    :return: additional information for the path (empty string if missing)
+    """
+    changelog_path = os.path.join(provider_package_path, "CHANGELOG.rst")
+    if os.path.isfile(changelog_path):
+        with open(changelog_path) as changelog_file:
+            return changelog_file.read()
+    else:
+        print(f"[red]ERROR: Missing ${changelog_path}[/]")
+        print("Please add the file with initial content:")
+        print()
+        syntax = Syntax(
```

Comment:
Would it make sense to extract this to a constant to move it outside of the function?